Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
```

Comment:
Why does this toml still have clippy mentioned. That's one of the main points of this PR - to upade CI scripts to not require clippy in-band. Pls go through lib.rs and every other file like tomls etc where changes need to be made. And in other similar PRs too.